dll/win32/shell32/lang/en-US.rc

```diff
@@ -805,6 +805,7 @@ BEGIN
     IDS_COPYERRORSAME "Cannot copy '%s': The source and destination file names are the same."
     IDS_MOVEERRORSUBFOLDER "Cannot move '%s': The destination folder is a subfolder of the source folder."
     IDS_COPYERRORSUBFOLDER "Cannot copy '%s': The destination folder is a subfolder of the source folder."
+    IDS_DISKFULL "There is not enough space on the disk."
```

Comment:
A media isn't necessarily a disk.

Reply:
Trying to copy file on net drive (Virtual Box shared folder) - it does not report ERROR_DISK_FULL error

Reply:
![xp_share](https://user-images.githubusercontent.com/12053509/145720620-9b38f92f-b33f-40c7-8ebe-cb4a3f940fa2.png)

same error on XP

Reply:
That's where comes wisdom: since a lot of people indeed complain that Windows' displayed errors are sometimes inaccurate, maybe one needs to manually check for that error and change it when being displayed.

Question: Does it return this error when there is no place anymore in the shared folder? or is it because it's read-only?

Reply:
no, it's not read-only - copying this file fine before error but with another name

Reply:
"maybe one needs to manually check for that error and change it when being displayed" - but how we can understand that ERROR_INVALID_PARAMETER is equal to ERROR_DISK_FULL but not any other error?

For read-only shared folder ROS and XP return ERROR_WRITE_PROTECT x19

Reply:
@katahiromz changed fixed message to FormatMessage of any error code